problem3.swift

```diff
@@ -14,6 +14,62 @@ while let inputNum = Double(readLine()!) {
     yData.append(inputNum)
 }
 
-
 // Write your linear regression code here
 
+//M
+//multiply xData by corresponding yData, store in  xyData THEN avg as xyCoAvg
+var xyData: [Double] = []
+for i in 0...(xData.count - 1){ //MULT xData value to corresponding yData, storing
```

Comment:
You can actually do for index in 0..<xData.count and that means the same thing as 0...(xData.count - 1) !